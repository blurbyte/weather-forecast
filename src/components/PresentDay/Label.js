import styled from 'styled-components';

const Description = styled.span`
  color: rgba(34, 34, 34, 0.5);
  margin-right: 0.4rem;
  text-transform: uppercase;
`;

export default Description;
