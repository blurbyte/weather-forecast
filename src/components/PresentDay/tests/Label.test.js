import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Label from '../Label';

test('renders correctly', () => {
  const component = renderer.create(<Label />);
  expect(component.toJSON()).toMatchSnapshot();
});
