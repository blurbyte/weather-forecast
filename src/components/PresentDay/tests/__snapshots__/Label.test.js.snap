// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  color: rgba(34,34,34,0.5);
  margin-right: 0.4rem;
  text-transform: uppercase;
}

<span
  className="c0"
/>
`;
